x-pack/plugins/apm/public/components/app/diagnostics/summary_tab/index.tsx

```diff
@@ -6,13 +6,30 @@
  */
 
 import React from 'react';
-import { EuiFlexGroup } from '@elastic/eui';
+import { EuiFlexGroup, EuiCallOut } from '@elastic/eui';
 import { ApmIntegrationPackageStatus } from './apm_integration_package_status';
 import { IndexTemplatesStatus } from './index_templates_status';
 import { FieldMappingStatus } from './indicies_status';
 import { DataStreamsStatus } from './data_streams_status';
+import { useDiagnosticsContext } from '../context/use_diagnostics';
 
 export function DiagnosticsSummary() {
+  const { diagnosticsBundle } = useDiagnosticsContext();
+
+  const isCrossCluster = Object.values(
+    diagnosticsBundle?.apmIndices ?? {}
+  ).some((indicies) => indicies.includes(':'));
+
+  if (isCrossCluster) {
+    return (
+      <EuiCallOut title="Cross cluster search not supported" color="warning">
```

Comment:
is this something we could implement in a future version of the diagnostics tool?

Reply:
Maybe. But I doubt it. Elasticsearch simply doesn't support it. I wasn't really aware of this but if you navigate to Stack Management it's quite limited what insights you have into remote clusters (data streams, index templates from remote clusters are all missing).